src/main/resources/application.properties

```diff
@@ -48,7 +48,6 @@ azure.storage.connection-string=${AZURE_STORAGE_CONNECTION_STRING:UseDevelopment
 azure.storage.table-name=${AZURE_STORAGE_TABLE_NAME:tools}
 
 # PostgreSQL Configuration for Quarkus
-quarkus.datasource.username=${POSTGRESQL_DB_URL:postgres}
-quarkus.datasource.password=${POSTGRESQL_DB_USERNAME:secret}
-quarkus.datasource.jdbc.url=${POSTGRESQL_DB_PASSWORD:jdbc:postgresql://localhost:5432/mydatabase}
-
+quarkus.datasource.jdbc.url=jdbc:postgresql://${POSTGRESQL_DB_HOST:localhost}:5432/${POSTGRESQL_DB_NAME:dev_database}?currentSchema=${POSTGRESQL_DB_SCHEMA:tool_service}
```

Comment:
should port also not be in .env?

Reply:
Ja dan zal dat moeten worden toegevoegd bij de platform-config, moet even bespreken met Sem.

Reply:
Ik zou t niet doen, want het blijft altijd hetzelfde. Is zelfde argument voor het `jdbc:postgresql://` stukje en `?currentSchem=`. Daan en ik hadden er een beetje over gediscussieerd en vonden dit wel een goede balans tussen legibility en verbosity. We hebben er basically voor gekozen om de dingen die veranderen in .env te doen, en de dingen waarbij we ervanuit kunnen gaan dat ze nooit veranderen en niet afhankelijk zijn van een organization policy gwn static uittypen.